connector/mqtt/vernemq/v2k-bridge/app/app.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @overview Initialize a v2k-client creating an Agent Messenger,
+ * then initialize it, also set log level for the service.
+ * @author Dojot
+ */
+
+const { Logger } = require('@dojot/microservice-sdk');
+
+const { app: appConfig } = require('./config');
+const AgentMessenger = require('./AgentMessenger');
+
+// Logger configuration
+Logger.setTransport('console', appConfig.logger.transports.console);
+Logger.setVerbose(appConfig.logger.verbose);
+
+const agentMessenger = new AgentMessenger();
+
+agentMessenger.init();
```

Comment:
Maybe you can wrapper this with a try/catch.

Reply:
Inside the `AgentMessenger.init` function I already have a then/catch in the promise, is it necessary to do a try/catch here too?

Reply:
If any other unhandled exception is thrown.